devicon.json

```diff
@@ -2535,6 +2535,31 @@
           }
         ]
     },
+    {
+        "name": "sqlalchemy",
+        "tags": [
+            "python",
+            "orm"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#333333",
+        "aliases": [
+          {
+            "base": "plain",
+            "alias": "original"
```

Comment:
You don't have to create an `alias` of the "plain" for the "original". We create aliases due to how the versions are used by the community. "plain" versions are guaranteed to be Icons while "original" aren't. Thus, people would expect that there's a class name like `.sqlalchemy-plain` to be available while don't think much of the `.sqlalchemy-original` class.

Since you already have the "plain" version, there is no need to create the "original" version. You can remove the alias object here.

Second thing: if you want to create a "plain-wordmark" version, you should put it here. This is applicable as long as the "original-wordmark" can be made into a font version.

```
"aliases": [
  {
    "base": "original-wordmark",    
    "alias": "plain-wordmark",
  }
]
```